### DISCUSSION TITLE
CentOS end of support

### DISCUSSION THREAD

**Discussion by greatapoc** [Security-Onion-Solutions/securityonion#2311] [General] [answered]
Any word on what the recent news about CentOS means for the project? Moving to one of the forks instead?

**Comment by dougburks** [comment, marked as answer]
Security Onion 2 will continue to support both CentOS 7 and Ubuntu 18.04 for the time being. We are monitoring this situation closely and when it comes time to migrate off of CentOS 7, we'll consider our options at that time including the new Rocky Linux (https://rockylinux.org)

**Reply by Masaya-A** [reply]
How about Oracle Linux 8?

**Reply by TOoSmOotH** [reply]
We will not be moving to Oracle Linux.

**Reply by Masaya-A** [reply]
May I ask reasons why not Oracle Linux?

**Reply by jonathanspw** [reply]
AlmaLinux has a higher adoption rate than Rocky so far.
